2-Draft-Accepted/RFCxxxx-DSC-for-PowerShell7.md

```diff
@@ -0,0 +1,117 @@
+
+## Motivation
+
+    As a developer of configuration solutions,
+    I can easily leverage DSC resources,
+    so that my solution can manage Windows.
+
+    As an ITPro,
+    I can directly invoke DSC resources,
+    so that I can build ad hoc cross platform configuration.
+
+## Invoke-DscResource
+
+About a year ago, we worked with [Gael Colas](https://twitter.com/gaelcolas) to author a RFC describing fundamental changes to 
+[`Invoke-DscResource`](https://docs.microsoft.com/powershell/module/psdesiredstateconfiguration/invoke-dscresource?view=powershell-7) in PowerShell 7.0.
+This enabled executing the Get, Set, and Test methods on DSC resources bypassing the LCM (Local Configuration Manager) on Windows and removing the
+need for OMI ([Open Management Infrastructure](https://github.com/Microsoft/omi)) on Linux.
+This was the start to enable cross platform DSC resources to be authored in PowerShell script (with some limitations).
+
+This feature was shipped as an [experimental feature](https://docs.microsoft.com/powershell/scripting/learn/experimental-features?view=powershell-7) in PowerShell 7.0.
+
+### Bring your own agent
+
+A significant change from existing DSC for Windows PowerShell is there will be no default agent (LCM) shipped with PowerShell 7.
+
+Instead, we want to make it easier to integrate with existing configuration agents like [Ansible](https://docs.ansible.com/ansible/latest/user_guide/windows_dsc.html),
+[Chef](https://docs.chef.io/resources/dsc_resource/), [Puppet](https://forge.puppet.com/puppetlabs/dsc),
+[Azure Policy Guest Configuration](https://docs.microsoft.com/azure/governance/policy/concepts/guest-configuration), and other popular configuration solutions.
+
+Custom agents can simply use the existing PowerShell API (or call out to `pwsh`) to directly invoke DSC resources.
+
+You can even simply write a PowerShell script as your agent that runs off TaskScheduler or cron, for example.
+
+Along with the lack of a default agent, there is no pre-defined push or pull model for deploying DSC.
+
+### Cross platform first
+
+DSC has supported both Windows and Linux, however, the [Linux support](https://docs.microsoft.com/powershell/scripting/dsc/getting-started/lnxGettingStarted?view=powershell-7)
+was dependent on [OMI](https://github.com/Microsoft/omi) and did not support authoring Linux DSC resources in PowerShell script making development
+inconsistent and much more difficult.
+
+DSC for PowerShell 7 will be cross platform having consistent capabilities and user experience on all platforms.
+
+### Moving away from WMI/OMI/MOF
+
+DSC on Windows was originally designed relying on [WMI](https://docs.microsoft.com/windows/win32/wmisdk/wmi-start-page) as a host on Windows,
+OMI as a host on Linux, and [MOF](https://docs.microsoft.com/windows/win32/wmisdk/managed-object-format--mof-) for configuratio and schema.
```

Comment:
typo: "for configuratio**n** and schema".
consider mentioning CIM is involved to convert PS Data Structures back in forth, even when invoking from PowerShell.

Reply:
will add some language about that